pkg/api/helpers.go

```diff
@@ -118,6 +118,12 @@ func NewDeleteOptions(grace int64) *DeleteOptions {
 	return &DeleteOptions{GracePeriodSeconds: &grace}
 }
 
+// NewListOptions returns a ListOptions indicating the resource should be
+// listed with at least given 'resourceVersion'.
+func NewListOptions(resourceVersion string) *ListOptions {
```

Comment:
Yes - initially I though it will be, but it doesn't seem to be that useful.
Removing it.